ovos_phal_connectivity_events/__init__.py

```diff
@@ -36,41 +38,71 @@ class ConnectivityEvents(PHALPlugin):
     def __init__(self, bus=None, config=None):
         super().__init__(bus=bus, name="ovos-PHAL-plugin-connectivity-events", config=config)
         self.sleep_time = 60
+        self.stopping = Event()
         self.state = ConnectivityState.UNKNOWN
         self.bus.on("ovos.PHAL.internet_check", self.handle_check)
-        self.bus.emit(Message("ovos.PHAL.internet_check"))
```

Comment:
this needs a self.start call since its a thread now

PHAL doesnt call start in case plugins want to do some checks and require conditions before that (eg, on some event)